Gemfile

```diff
@@ -18,7 +18,7 @@ gem 'puma'
 gem 'samwise', '~> 0.4.0'
 gem 'octokit', '~> 4.0'
 gem 'rack-cors', require: 'rack/cors'
-gem 'active_model_serializers'
+gem 'active_model_serializers', '~> 0.9.5'
```

Comment:
GREAT call -- I spent about 30 mins trying to upgrade to `0.10` and it was such a disaster I gave up